test/e2e/storage/vsphere/vsphere_utils.go

```diff
@@ -156,45 +155,21 @@ func waitForVSphereDiskToDetach(volumePath string, nodeName string) error {
 
 // function to create vsphere volume spec with given VMDK volume path, Reclaim Policy and labels
 func getVSpherePersistentVolumeSpec(volumePath string, persistentVolumeReclaimPolicy v1.PersistentVolumeReclaimPolicy, labels map[string]string) *v1.PersistentVolume {
-	var (
-		pvConfig framework.PersistentVolumeConfig
-		pv       *v1.PersistentVolume
-		claimRef *v1.ObjectReference
-	)
-	pvConfig = framework.PersistentVolumeConfig{
+	return framework.MakePersistentVolume(framework.PersistentVolumeConfig{
 		NamePrefix: "vspherepv-",
 		PVSource: v1.PersistentVolumeSource{
 			VsphereVolume: &v1.VsphereVirtualDiskVolumeSource{
 				VolumePath: volumePath,
 				FSType:     "ext4",
 			},
 		},
-		Prebind: nil,
-	}
-
-	pv = &v1.PersistentVolume{
-		ObjectMeta: metav1.ObjectMeta{
-			GenerateName: pvConfig.NamePrefix,
-			Annotations: map[string]string{
-				util.VolumeGidAnnotationKey: "777",
```

Comment:
Do you think any tests will complain about this being removed?

Reply:
all `PV`'s in the test are defaulted to that https://github.com/kubernetes/kubernetes/pull/80543/files#diff-420013911dac4a3b0f16035b6e90a571R624